api/V1/Visits.ts

```diff
@@ -50,19 +56,23 @@ function getTrackTag(trackTags: TrackTag[], userID: number): TrackTag | null {
   }
   const manualTags = trackTags.filter((tag) => tag.automatic == false);
   if (manualTags.length > 0) {
-    const userTag = manualTags.find((tag) => tag.UserId == userID);
-    if (userTag) {
-      return userTag;
-    } else {
-      return manualTags[0];
+    const animalTags = manualTags.filter(
+      (tag) => !nonAnimalTags.includes(tag.what)
+    );
+    const uniqueTags = new Set(animalTags.map((tag) => tag.what));
+    if (uniqueTags.size > 1) {
```

Comment:
Do we want to return a conflict now, or after we analyse the whole visit?   We are losing data.   It is a rare suituation so I guess it doesn't matter much. Do we want to say what the conflict is between?